eventually/team/models.py

```diff
@@ -0,0 +1,169 @@
+"""
+Team model
+=======================
+
+This module implements class that represents the team entity.
+"""
+from django.db import models
+
+
+class Team(models.Model):
+
+    """
+    .. class:: Team
+    Describing of team entity.
+    Attributes:
+    ===========
+
+        :param name: Name of the certain team.
+        :type name: string
+
+        :param description: Describing core information of the team.
+        :type description: string
+
+        :param image: Describing link to image of the team.
+        :type description: string
+
+        :param create_date: The date when the
+                            certain team was created.
+        :type: create_date: datetime
+
+        :param update_date: The date when
+                    the certain team was last time edited.
+        :type: create_date: datetime
+    """
+    name = models.CharField(max_length=30)
+    description = models.TextField(max_length=1024, blank=True)
```

Comment:
If you specify a max_length attribute, it will be reflected in the Textarea widget of the auto-generated form field. However it is not enforced at the model or database level. Use a CharField for that

Reply:
Done

Reply:
don`t forget to make migrations